substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/ProgressReporter.java

```diff
@@ -77,6 +77,7 @@
     private static final int CHARACTERS_PER_LINE;
     private static final int PROGRESS_BAR_START = 30;
     private static final boolean IS_CI = System.console() == null || System.getenv("CI") != null;
+    boolean IS_DUMB_TERM = "dumb".equals(System.getenv("TERM"));
```

Comment:
It seems `"unknown"` is another synonym for `"dumb"` that we should probably recognize here.
See https://stackoverflow.com/questions/39001517/features-obligatory-for-term-dumb-terminal

Also if `TERM` is not set at all (or empty) we should also interpret that as `IS_DUMB_TERM`.

Reply:
Yes, I agree we should treat all 3 cases as IS_DUMB_TERM. Actually there are four cases because we need to allow for System.getenv returning null, "", "dumb" or "unknown". I have pushed an update.